qa-pipelines/qa-pipeline.yml.erb

```diff
@@ -227,26 +255,38 @@ jobs:
     # We intentionally don't put the teardown and pool release steps in an ensure
     # block, so that when tests fail we have a chance of examining why things are
     # failing.
-    <% if enable_cf_teardown %>
+    <% if enable_cf_teardown && !enable_terraform %>
     - task: cf-teardown
       file: ci/qa-pipelines/tasks/cf-teardown.yml
       timeout: 1h
       params: *common-task-params
-    <% end %> # enable_cf_teardown
+    <% end %> # enable_cf_teardown and not enable_terraform
```

Comment:
`and` and `&&` are not operationally equivalent in ruby, so let's stick with `&&` to reflect the condition being closed

Reply:
Change requested